launcher/modplatform/flame/FlameInstanceCreationTask.cpp

```diff
@@ -404,6 +411,38 @@ void FlameCreationTask::idResolverSucceeded(QEventLoop& loop)
     }
 }
 
+/// @brief copy the matched blocked mods to the instance staging area
+/// @param blocked_mods list of the blocked mods and their matched paths
+void FlameCreationTask::copyBlockedMods(QList<BlockedMod> blocked_mods) {
```

Comment:
```suggestion
void FlameCreationTask::copyBlockedMods(QList<BlockedMod> const& blocked_mods) {
```

note that this may create problems with the iterators you're using inside this function. however, you're not really mutating the `QList`, and this way we'll prevent a deep copy from happening if you access it in a non-const way (also, it may be better to use a range-for loop instead of a Java style one down there)

Reply:
good catch. I actually add it marked as `const` at one point but had to remove it during debugging for a reason I forgot, it's no longer necessary not to now though.